### PR TITLE
feat: Consolidate Cells

### PR DESCRIPTION
**Pull request by devchenyan**
issue: https://github.com/Magickbase/neuron-public-issues/issues/368

https://github.com/nervosnetwork/neuron/assets/11756869/73bb037f-7a28-40ab-bf23-5af2449f3958

### CODE REVIEW

**Comment by Danie0918** [comment]
@yanguoyu @homura Please have a review.

**Review by yanguoyu — APPROVED** [review]

File: packages/neuron-ui/src/locales/en.json [hunk 1]
```diff
@@ -1269,12 +1269,17 @@
         "title": "Consume Cell",
         "warn-consume": "Consume cell will not keep any data and use all of the ckb, please ensure that no important data needs to be retained"
       },
+      "cell-consolidate-dialog": {
+        "title": "Consolidate Cells",
+        "warn-consume": "All selected cells will be consolidated into a new cell, make sure that no cells containing important data are selected."
```
[the remark this answers is not in the document]
Comment (devchenyan) [reply packages/neuron-ui/src/locales/en.json:1274]: @Danie0918 please confirm whether `All selected cells will be consolidated into a new cell, make sure that no cells containing important data are selected.` needs to be changed to `All selected cells will be consolidated into a new cell, make sure that selected cells do not contain important data.`

image

[the remark this answers is not in the document]
Comment (Danie0918) [reply packages/neuron-ui/src/locales/en.json:1274]: > @Danie0918 please confirm whether `All selected cells will be consolidated into a new cell, make sure that no cells containing important data are selected.` needs to be changed to `All selected cells will be consolidated into a new cell, make sure that selected cells do not contain important data.`
> 
> image

OK, It's better to change.

**Review by homura — APPROVED** [review]

**Comment by silySuper** [comment]
When window is smallest,send to address can not see whole address,and reset tip location is abnormal
截屏2024-06-04 14 48 24

**Review by silySuper — APPROVED** [review]

**Pull request merged**